doc/source/changes/version_0_25.rst.inc

```diff
@@ -1,6 +1,9 @@
 ﻿New features
 ------------
 
+* viewer function view, edit and compare require to have package larray-editor installed.
```

Comment:
I would write something like:

viewer functions (view, edit and compare) have been moved to the separate `larray-editor` package, which needs to be installed separately, unless you are using `larrayenv`.